src/_posts/photosphere-viewer.md

```diff
@@ -0,0 +1,43 @@
+---
+title: 360&deg; Photosphere Viewer
+date: 2016-1-22
+author: twitter|ideaspacevr
+layout: blog
+image:
+  href: https://www.ideaspacevr.org/themes/webvr-photosphere-viewer-equirectangular-images
+  src: photosphere-viewer.png
+---
+
+> Chris from [IdeaSpaceVR](https://www.ideaspacevr.org/) has built an interactive 360-degree photosphere viewer. Space bar toggles a menu system, and a gaze-based cursor selects photos.
+
+Recently I installed the Cardboard Camera app on my phone. It is a 360-degree photosphere VR app for Google Cardboard including a nice menu in VR which allows one to easily select photos from a list. The photosphere is showing photos with a so-called equirectangular projection (did not hear about that term before, as I am not a photo nerd ;-) ).
+
+Since I knew that A-Frame had been released I thought of implementing this app in A-Frame, as I thought it might be quite easy to do, AND I would learn coding in A-Frame.
+
+<!-- more -->
+
+After a few hours of playing around I realised that
+
+(a) yes it is easy to implement a photosphere BUT
+(b) implementing a decent VR menu could not be done by just using A-Frame tags.
+
+Which lead me to implement my first component for A-Frame. As it turned out, I needed actually two components:
+
+The first component handles the VR photo menu (showing the menu, hiding the menu, position it in space). This component acts as a container for the photo thumbnails. You can check out the [source code here](https://github.com/IdeaSpaceVR/aframe-360-degree-photosphere/blob/master/js/components/isvr_photosphere_menu.js). I have to admit that I was not able to write the code for positioning the menu in space. So I turned to [Stackoverflow](http://stackoverflow.com/questions/34447119/positioning-a-three-js-object-in-front-of-the-camera-but-not-tied-to-the-camera) which helped me out on that one (again).
```

Comment:
oopsies hehe:

![image](https://cloud.githubusercontent.com/assets/203725/14481735/c119936e-00ea-11e6-8a5c-e8672b91c951.png)

feel free to remove [change the selector of the GitHub link styles](https://github.com/aframevr/aframe-site/blob/master/themes/aframe/source/css/secondary.styl#L240-L267)

drop `p [href*=github][href*="/blob/"]:only-child` but keep `.github-file-link`.